blog.html

```diff
@@ -22,6 +22,286 @@
+                    deployment and management, eliminating the complexity of maintaining a separate ZooKeeper
+                    ensemble. This change significantly reduces operational overhead, enhances scalability, and
+                    streamlines administrative tasks. We want to take this as an opportunity to express our gratitude
+                    to the ZooKeeper community and say, thank you! ZooKeeper was the backbone of Kafka for
+                    more than 10 years, and it did serve Kafka very well. Kafka would most likely not be what it is
+                    today without it. We don’t take this for granted and highly appreciate all the hard work the
+                    community invested to build ZooKeeper. Thank you!
+                </p>
+                <p>
+                    Kafka 4.0 also brings the general availability of <a href="https://cwiki.apache.org/confluence/x/HhD1D">KIP-848</a> introducing a powerful new consumer
+                    group protocol designed to dramatically improve rebalance performance. This optimization
+                    significantly reduces downtime and latency, enhancing the reliability and responsiveness of
+                    consumer groups, especially in large-scale deployments.
+                </p>
+                <p>
+                    Additionally, we are excited to offer <a href="https://cwiki.apache.org/confluence/x/i4ogF">early access</a> to
+                    Queues for Kafka (<a href="https://cwiki.apache.org/confluence/x/4hA0Dw">KIP-932</a>), enabling Kafka to support traditional queue semantics directly. This
+                    feature extends Kafka’s versatility, making it an ideal messaging platform for a wider range of
+                    use cases, particularly those requiring point-to-point messaging patterns.
+                </p>
+                <p>
+                    These highlights illustrate Kafka's continued evolution to meet the demands of modern,
+                    scalable, and efficient data streaming and messaging architectures.
+                </p>
+                
+                <h3>Kafka Broker, Controller, Producer, Consumer and Admin Client</h3>
+
+                <ul>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/HhD1D">KIP-848: The Next Generation of the Consumer Rebalance Protocol</a>
+                        <br>
+                        Apache Kafka also says goodbye to “stop-the-world” rebalances with the general availability of the
+                        next generation of the consumer rebalance protocol. It increases the stability and the performance
+                        of consumer groups while simplifying clients. The new protocol is enabled by default on the server
+                        side. Consumers must opt-in by setting <code>group.protocol=consumer</code>.
```

Comment:
Is there a plan to make this the client-side default in 4.x? (e.g 4.1?) Perhaps worth mentioning if so

Reply:
agreed, not sure if an exception would work for this feature. makes sense to not mention if there isn't a clear date yet!

Reply:
I will write a KIP for this soon. TL/DR: We will change the default in 5.0 and remove the option in 6.0.